code/portal_detail_display.js

```diff
@@ -98,12 +98,12 @@ window.renderPortalDetails = function(guid) {
       
       $('<h3>').attr({class:'title'}).text(title),
 
-      $('<span>').attr({
-        class: 'close',
-        title: 'Close [w]',
-        onclick:'renderPortalDetails(null); if(isSmartphone()) show("map");',
-        accesskey: 'w'
-      }).text('X'),
+      $('<span>', {class: 'close', title: 'Close [w]', accesskey: 'w' })
+        .click(function() {
```

Comment:
maybe use .on('click'... ?

Reply:
[click](https://api.jquery.com/click/) method is just shortcut for `on.('click'`, so it has the same action.

Reply:
Ok, I mixed up with the old version of jquery